custom_components/wyzeapi/__init__.py

```diff
@@ -87,6 +82,16 @@ async def async_setup_entry(hass: HomeAssistant, config_entry: ConfigEntry) -> b
 
     hass.data.setdefault(DOMAIN, {})
 
+    # Store a uuid for the notification toggle if it doesn't exist to get away from storing it in an ini file
+    if not config_entry.data.get(UUID):
+        _LOGGER.debug("No Existing UUID found for %s", DOMAIN)
```

Comment:
Can we check to see if there is an existing wyze_config.ini file here and load from it if it exists?

Reply:
We could, but that would require us to keep the ini reading code in place in addition to this code. 

During my testing, it didn't seem like the UUID actually *needs* to be persisted to toggle the notification setting.

Reply:
This has hopefully be addressed in the latest commit on this PR.